import-service/src/functions/importProductsFile/index.ts

```diff
@@ -17,6 +18,12 @@ export default {
                     200: 'Successful operation',
                     500: 'Server error',
                 },
+                authorizer: {
+                    name: 'basicAuthorizer',
+                    arn: 'arn:aws:lambda:eu-west-1:147259031518:function:authorization-service-dev-basicAuthorizerHandler',
+                    type: 'TOKEN',
```

Comment:
Your authorizer lambda returns "deny" response if type !== "REQUEST", so any request with Authorization header to this lambda will get 403 error